frame.go

```diff
@@ -67,16 +79,52 @@ func (f *Frame) Draw() {
 	PushAttributes()
 	defer PopAttributes()
 
+	x, y, w, h := f.Clipper()
+
+	if f.scrollable {
+
+		_, fy := f.Pos()
+		_, fh := f.Size()
+		_, fpy := f.Paddings()
+
+		var dist float64
+		prop := 0
+		ctrl := ActiveControl(f)
+
+		if ctrl != nil {
+			var frameProp float64
+
+			_, ty := ctrl.Pos()
+
+			dist = (float64(fy) + float64(fpy)) - float64(ty)
+			dist = math.Sqrt(dist * dist)
+
+			if dist > 0 {
+				frameProp = (dist * 100) / float64(fh)
+			}
+
+			if frameProp > 0 {
+				prop = int(math.Round((float64(h) / (100 / frameProp))))
+			}
+
+			f.lastScrollProp = prop
+		}
+
+		DrawScrollBar(x+w, y, 1, h, f.lastScrollProp)
+	}
+
+	fg, bg := RealColor(f.fg, f.Style(), ColorViewText), RealColor(f.bg, f.style, ColorViewBack)
```

Comment:
Why one uses `f.Style()` while another one `f.style`?

Reply:
No reason at all, just a careless and inconsistent programmer :)